config/metadata/configMeta.yaml

```
@@ -816,15 +816,15 @@ groups:
         v1name: Peers
         type: stringarray
         valuetype: stringarray
-        example: "192.168.1.11:8081,192.168.1.12:8081"
+        example: "http://192.168.1.11:8081,http://192.168.1.12:8081"
         reload: false
         validations:
           - type: elementType
-            arg: hostport
+            arg: url
         summary: is the list of peers to use when Type is "file", excluding self.
         description: >
           This list is ignored when Type is "redis". The format is a list of
-          strings of the form "host:port".
+          strings of the form "schema://host:port".
```

```suggestion
          strings of the form "scheme://host:port".
```
Please make this change and regenerate. For URLs, it's "scheme" not "schema". Yay English. :)